charts/sftd/values.yaml

```diff
@@ -83,10 +83,9 @@ tls: {}
 joinCall:
   replicaCount: 3
   image:
-    repository: nginxinc/nginx-unprivileged
+    repository: docker.io/bitnami/nginx
```

Comment:
I assume that we have the appropriate `imagePullSecrets` set up?

Reply:
the join-call deployment support imagePullSecrets, if that's what you mean.
this concrete image is public

Reply:
I'm asking because since DockerHub started imposing rate limits for everything, it turned out to be a good practice to only do authenticated pulls (since unauthenticated pulls have quite a low rate limit per IP).

At least I already got bitten by that rate limit

Reply:
Ah I see. The chart already supports imagePullSecrets, so we can do this. It would have to be done at the deployment config, not in the defaults of the helm chart.